firmware/brake/kia_soul_ps/brake_control_module.ino

```diff
@@ -142,66 +142,115 @@ int16_t solenoid_left_actuation_duty_min;
+// *****************************************************
+// Function:    convert_to_voltage
+// 
+// Purpose:     Convert the ADC reading
+//				(which goes from 0-1023)
+//				to a voltage (0 - 5V)
+// 
+// Returns:     float - volts
+// 
+// Parameters:  int16_t - ADC reading 
+// 
+// *****************************************************
 float convert_to_voltage( int16_t input )
 {
     return input * ( 5.0 / 1023.0 );
 }
 
-
+// *****************************************************
 // MOSFET pin (digital) definitions ( MOSFETs control the solenoids )
-// pins are not perfectly sequential because the clock frequency of certain pins is different.
+//
+// Pins are not perfectly sequential because the clock frequency of certain pins is different.
+//
+// *****************************************************
 // Duty cycles of pins 3 and 5 controlled by timer 3 (TCCR3B)
-const byte PIN_SLAFL = 3;      // front left actuation
-const byte PIN_SLAFR = 5;      // front right actuation
+const uint8_t PIN_SLAFL = 3;      // front left actuation
+const uint8_t PIN_SLAFR = 5;      // front right actuation
 // Duty cycles of pins 6, 7, and 8 controlled by timer 4 (TCCR4B)
-const byte PIN_SLRFL = 6;      // front left return
-const byte PIN_SLRFR = 7;      // front right return
-const byte PIN_SMC = 8;      // master cylinder solenoids (two of them)
-
-const byte PIN_PUMP = 9;     // accumulator pump motor
+const uint8_t PIN_SLRFL = 6;      // front left return
+const uint8_t PIN_SLRFR = 7;      // front right return
+const uint8_t PIN_SMC = 8;      // master cylinder solenoids (two of them)
 
+const uint8_t PIN_PUMP = 9;     // accumulator pump motor
 
 // brake spoofer relay pin definitions
-const byte PIN_BRAKE_SWITCH_1 = 48;
-const byte PIN_BRAKE_SWITCH_2 = 49;
-
+const uint8_t PIN_BRAKE_SWITCH_1 = 48;
+const uint8_t PIN_BRAKE_SWITCH_2 = 49;
 
 // sensor pin (analog) definitions
-const byte PIN_PACC = 9;       // pressure accumulator sensor
-const byte PIN_PMC1 = 10;      // pressure master cylinder sensor 1
-const byte PIN_PMC2 = 11;      // pressure master cylinder sensor 2
-const byte PIN_PRL = 12;       // pressure rear left sensor
-const byte PIN_PFR = 13;       // pressure front right sensor
-const byte PIN_PFL = 14;       // pressure front left sensor
-const byte PIN_PRR = 15;       // pressure rear right sensor
+const uint8_t PIN_PACC = 9;       // pressure accumulator sensor
+const uint8_t PIN_PMC1 = 10;      // pressure master cylinder sensor 1
+const uint8_t PIN_PMC2 = 11;      // pressure master cylinder sensor 2
+const uint8_t PIN_PRL = 12;       // pressure rear left sensor
+const uint8_t PIN_PFR = 13;       // pressure front right sensor
+const uint8_t PIN_PFL = 14;       // pressure front left sensor
+const uint8_t PIN_PRR = 15;       // pressure rear right sensor
 
 // the following are guesses, these need to be debugged/researched
 const double ZERO_PRESSURE = 0.48;        // The voltage the sensors read when no pressure is present
```

Comment:
I looked up instruction execution timing and I think we should stay away from doubles.  They just take too long.  Please change these to floats.